docs/installing-solidity.rst

```diff
@@ -100,6 +100,12 @@ Now add all the rest:
     sudo add-apt-repository -y ppa:ethereum/ethereum-dev
     sudo apt-get -y update
     sudo apt-get -y upgrade
+    
+Then install Solidity:
```

Comment:
This section is about "building from source", so you would not usually run this here.